templates/project/.github/workflows/publish.yml

```diff
@@ -18,13 +18,17 @@ jobs:
         fetch-depth: 2
     - name: Build esp32 OTA binary
       run: |
-        bash -c '. /home/testbot/.bash_profile ; git config --global --add safe.directory "$ESP_ROOT"/esp-idf && make build-esp32-ota'
-        cp target/xtensa-esp32-espidf/project-ota.bin project-ota.bin
+        bash -c '. /home/testbot/.bash_profile ; git config --global --add safe.directory "$ESP_ROOT"/esp-idf && make build-esp32-bin build-esp32-ota'
+        cp target/xtensa-esp32-espidf/release/{{project-name}} project.elf
+        cp target/xtensa-esp32-espidf/release/{{project-name}}.bin project.bin
```

Comment:
We need to know the project name here, since the ELF binary will have that name (we can control the names for the `.bin` files, but as far as I know the ELF file is auto-determined by cargo. So, now this file needs substitution. See below for the sad sad effects.

Reply:
does that mean we have to leave the asset name as project.elf?

Reply:
No, we can call this whatever we want. I can easily interpolate the project name into the files here. If we are doing that, I think is probably worthwhile to put the tag in as well, so:

`project-x.y.z.elf`
`project-x.y.z.bin`
`project-ota-x.y.z.bin`

Reply:
I think this is worth doing right now

Reply:
I've put the project name in, but left the version out, for now. We can revisit later.